.gitignore

```diff
@@ -22,5 +22,4 @@ nb*.xml
 **/*.iml
 
 
-
```

Comment:
Unnecessary change.
Since `.flattened-pom.xml` is already tracked only (re-)adding to gitignore won't work. It must be explicitly deleted.

For reference see:
- https://git-scm.com/docs/gitignore#_description
- https://git-scm.com/docs/gitignore#_notes
- https://git-scm.com/docs/git-rm#Documentation/git-rm.txt---cached